src/components/Checkbox.js

```diff
@@ -13,9 +13,9 @@ export default class Checkbox extends React.Component {
   }
   render() {
     return (
-        <div>
-          <input type="checkbox" checked={this.state.checked} onChange={() => this.toggle(this.props.Name)}/>
-          <label>{this.props.label ? this.props.label : null}</label>
+        <div className="checkbox--MAIN">
+          <input className="checkbox--checkbox" type="checkbox" checked={this.state.checked} onChange={() => this.toggle(this.props.Name)}/>
```

Comment:
The className is just something I added, but I don't think I've applied styles to `checkbox--checkbox` yet.